src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -37,6 +37,9 @@ namespace Microsoft.ML.Transforms
     /// <include file='doc.xml' path='doc/members/member[@name="TensorflowTransformer"]/*' />
     public sealed class TensorFlowTransformer : RowToRowTransformerBase
     {
+        /// <summary>
+        /// The settings for the <see cref="TensorFlowTransformer"/>.
+        /// </summary>
         public sealed class Options : TransformInputBase
```

Comment:
>public [](start = 8, length = 6)

How come you internalize options in OnnxTransform but not here?
I remember how I copypasted code from one transform to another, did they diverge somehow?

Reply:
In OnnxTransform we have all the parameters explicitly in the arguments of the constructors, and we don't use Options in the extensions. In TensorFlowTransforms instead there are many more parameters and we don' have a constructor that with all of them, so we still have a constructor with Options.

---
In reply to: [254139791](https://github.com/dotnet/machinelearning/pull/2367#discussion_r254139791) [](ancestors = 254139791)